pandas/tseries/period.py

```diff
@@ -281,6 +281,16 @@ def __contains__(self, key):
             return False
         return key.ordinal in self._engine
 
+    def __array_wrap__(self, result, context=None):
```

Comment:
Yes, because this is a valid freq shift, ``np.array`` should work as the same. 

```
pd.PeriodIndex(['2011', '2012'], freq='A') + 1
# PeriodIndex(['2012', '2013'], dtype='int64', freq='A-DEC')
```

And #10744 has been done expecting arithmetic using ``np.array`` works.